packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -262,10 +264,17 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     maxDominantSpeakers: MAX_AUDIO_DOMINANT_SPEAKERS
   });
 
+  /* @conditional-compile-remove(rooms) */
+  const permissions = _usePermissions();
```

Comment:
Similar to the comment on the change file, (not in this PR) can we change this hook to be useRolePermissions (or just useRole) - reason being I read this as if the browser has blocked permissions to the camera

Reply:
useRolePermissions will be good until we start getting Teams and Rooms roles in the mix.  (Possible suggestion "useRoomRolePermissions")
 
 I am less concerned about the name since by the time we hit GA this information will be available through calling sdk (and our selectors) and we won't need to bring the hooks from public preview -> GA with us.